packages/drivers/odsp-driver/src/odspDocumentService.ts

```diff
@@ -246,7 +246,7 @@ export class OdspDocumentService implements IDocumentService {
                 const connection = await this.connectToDeltaStreamWithRetry(
                     websocketEndpoint.tenantId,
                     websocketEndpoint.id,
-                    webSocketToken,
+                    webSocketToken || websocketEndpoint.socketToken,
```

Comment:
Please add the comment on why this is right logic - it gets lost and folks will make mistake when refactoring.

Reply:
Added comments